Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/) for Maemo Leste/hildon-common-strings-l10n.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/maemo-leste/-/hildon-common-strings-l10n/horizontal-auto.svg)

fitojb committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/ca/

fitojb committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/ca/

fitojb commented:
@maemo-leste-translations If you haven’t already, please enable commit squashing by author on Weblate, to reduce the amount of generated commits.

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/es/

parazyd commented:
@fitojb Is there some simple way to install the addons for all repositories on weblate? Going one-by-one is not ideal really.

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/nb_NO/

SuperMarioSF committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/zh_Hans/

rhld16 committed:
Translated using Weblate (Welsh)

Currently translated at 100.0% (243 of 243 strings)

Translation: Maemo Leste/hildon-common-strings-l10n
Translate-URL: https://hosted.weblate.org/projects/maemo-leste/hildon-common-strings-l10n/cy/
